### ISSUE TITLE
Tree (Browse)

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/portal#27]
https://www.dev.catalogue.life/data/browse.html

![image](https://user-images.githubusercontent.com/28574252/93227170-b4fe9080-f739-11ea-9e6e-e193a15640d6.png)
and
https://www.dev.catalogue.life/data/browse.html?taxonKey=061950e4-9782-4d1a-9c87-dcf375788e6b 

**Essential corrections needed:**

- [ ] Please, take off GSD logos from the Tree. Aliases/GSDShortNames are very much informative and helpful (incl. copy&paste of tree branch in Word doc) 

- [x] Please add actual species counts in the Tree

- [x] Please apply following style for presentation of actual species counts and estimates:
Phylum Annelida • 14,399 of 17,388 est. living spp (83%); 523 †spp
We cannot use word "described" in stats, because CoL never accounted “ever described species” in its estimates.

- [x] Please, add phrase "multiple providers" for branches where there are many GSDs (i.e. 1+, or 2+)  

- [x] It would be useful, if names appear in the Tree with authorstrings. It would be also useful to allow users customize a view “with or without” authorstrings.

- [x] Switch off option in AC interface for statistics, providers & extinct taxa was useful for those users who need a light view of the tree (Check boxes “Show statistics” “Show providers” “Show extinct taxa (†)” and new "Show authorstrings")

**Comment by mdoering** [comment]
This needs discussion. 
 - I prefer the tree with logos.
 - "multiple providers": we do show that in the assembly tree. But I would prefer if we standardize our terminology and not use GSD, provider, source, constituent, contributing dataset in various places. To avoid this discussion I would suggest to just use  "multiple" in the tree - at least for now.
 - remove "described": agree
 - stats: we dont have them available in the API now. If we do, I would be in favour to not mix them with the estimations as its done in the old UI, I found this rather hard to read. What about: `1.296.192 (†60.139) spp * estimation 1.525.728

Additionally:
 - [ ] Show reference of estimation. There was an info icon before that showed the source but also the stats again and a list of all sources by name. Should simply we do the same?
 - [ ] toggle to turn on/off the stats. I would not use 3 different toggles like we have in the old UI, but instead show all or nothing with the info icon always being present. 
 - [ ] The extinct toggle removed not only the counts but also removed the extinct taxa from the tree so you only browse living taxa. If we want to keep this feature this needs added to the [Tree API. This had been requested by GBIF users](https://github.com/CatalogueOfLife/backend/issues/837) already, so maybe we should?

**Comment by mdoering** [comment]
For species counts we have a dedicated issue: https://github.com/CatalogueOfLife/portal/issues/23

**Comment by dhobern** [comment]
For me, the logos are distracting noise, especially when they repeat for massive numbers of children. But I'm not strongly bothered either way.

**Comment by mdoering** [comment]
Extinct dagger to be used in front of names, to be supplied by the Tree API label method:
<img width="458" alt="Screenshot 2020-09-16 at 10 04 40" src="https://user-images.githubusercontent.com/327505/93436118-a3002780-f8ca-11ea-950a-551ad9477ddb.png">

**Comment by mdoering** [comment]
Taxon Info box in the old interface:

<img width="1037" alt="Screenshot 2020-09-16 at 13 26 22" src="https://user-images.githubusercontent.com/327505/93436122-a5628180-f8ca-11ea-8638-6cc0939e9f82.png">

**Comment by mdoering** [comment]
@thomasstjerne I have added a new `extinct` query param to the Tree API that filters out extinct taxa if `extinct=false` is given. It defaults to true, i.e. it shows both living and extinct.

**Comment by thomasstjerne** [comment]
> Taxon Info box in the old interface

I think it is missing because of https://github.com/CatalogueOfLife/backend/issues/866

**Comment by thomasstjerne** [comment]
> @thomasstjerne I have added a new `extinct` query param to the Tree API that filters out extinct taxa if `extinct=false` is given. It defaults to true, i.e. it shows both living and extinct. Available on dev shortly

It seems that `extinct=false` actually filters out all higher taxa (i.e. not marked explicitly with `extinct=false`)

**Comment by mdoering** [comment]
do you have an example taxon at hand? Animalia is the same:
http://api.catalogue.life/dataset/3LR/tree/061950e4-9782-4d1a-9c87-dcf375788e6b/children?extinct=true
http://api.catalogue.life/dataset/3LR/tree/061950e4-9782-4d1a-9c87-dcf375788e6b/children?extinct=false

**Comment by thomasstjerne** [comment]
Ahh, sorry - wrong issue. This is in the nameusage search 
https://data.dev.catalogue.life/dataset/1008/names?extinct=false&facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&limit=50&offset=0&sortBy=taxonomic

**Comment by mdoering** [comment]
Created a new issue for extinct. Looks like the API does the right thing, but data is the problem.
Let's continue the discussion on that issue https://github.com/CatalogueOfLife/backend/issues/870

**Comment by yroskov** [comment]
>>    guess we could make it a primitive boolean with just 2 states yes/no and simply default to not extinct in case the flag is not explicitly set? Is it intuitive or useful for a user to deal with nulls, i.e. unknown?

Only extinct species and higher taxa with all extinct children species should get "Extinct" comment on species page and dagger in the Tree, etc.. Neither extant, nor taxa with status "unknown" should get any comments in public view.

**Comment by mdoering** [comment]
@chantalhuijbers @thomasstjerne @yroskov could you summarize where we are with this collection of tree issues?
Please keep the species counts separate from this in its issue #23 and consider breaking this up into individual issues if they can/should be addressed independently

**Comment by mdoering** [comment]
I don't mind to keep or take off the logos. I think @dhobern also preferred the tree without them.
